prow/plugins/config_test.go

```diff
@@ -1137,3 +1137,83 @@ func TestStatesMatch(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateConfigUpdater(t *testing.T) {
+	testCases := []struct {
+		name        string
+		cu          *ConfigUpdater
+		expected    error
+		expectedMsg string
+	}{
+		{
+			name: "same key of different cms in the same ns",
```

Comment:
@petr-muller 
Even this case cannot pass right now.
It is stricter than we discussed.

Let me try to fix those 2 cases.

Reply:
Done.